common/config.gni

```diff
@@ -77,6 +77,10 @@ if (is_ios || is_mac || is_android || is_win) {
   feature_defines_list += [ "SHOREBIRD_PLATFORM_SUPPORTED=1" ]
 }
 
+if (is_ios) {
+  feature_defines_list += [ "SHOREBIRD_USE_LINKER=1" ]
```

Comment:
I'd probably call this USE_INTERPRETER but yeah YES, love the idea!

Reply:
Done